.github/workflows/deployment.yml

```
@@ -6,19 +6,18 @@ on:
   # You can manually trigger a deployment on GitHub.com
   # https://github.blog/changelog/2020-07-06-github-actions-manual-triggers-with-workflow_dispatch/
   schedule:
-    - cron: '31 13 * * 3'
+    - cron: '42 4 * * 4' # At 04:42 every Thursday
 
 jobs:
   Build:
     outputs:
-      created: ${{ steps.daily-version.outputs.created }}
+      shouldDeploy: ${{ steps.daily-version.outputs.created }} || github.event_name == 'workflow_dispatch'
```

Not sure if this is right or if it should be
```suggestion
      shouldDeploy: ${{ steps.daily-version.outputs.created || github.event_name == 'workflow_dispatch' }}
```